### PR TITLE
Make the reactive SQL clients tests more future proof

### PR DESCRIPTION
**Pull request by gsmet**
@vsevel hopefully, this will help.

You will have to test your things with:
- `-Dtest-postgresql -Ddocker` + use ${reactive-postgres.url} in your config file
- `-Dtest-mariadb -Ddocker` + use ${reactive-mysql.url} in your config file

Feel free to ping me if it doesn't help but I think you should be able to have real tests in the deployment module with these commits in.

/cc @geoand since we discussed it with you too

### CODE REVIEW

**Review by geoand — APPROVED** [review]
LGTM!

File: extensions/reactive-pg-client/deployment/pom.xml [hunk 1]
```diff
@@ -15,6 +15,11 @@
 
     <name>Quarkus - Reactive PostgreSQL Client - Deployment</name>
 
+    <properties>
+        <postgres.image>postgres:10.5</postgres.image>
+        <reactive-postgres.url>vertx-reactive:postgresql:///hibernate_orm_test</reactive-postgres.url>
```
Comment (vsevel) [inline extensions/reactive-pg-client/deployment/pom.xml:20]: out of curiosity, why do we need 2 `reactive-postgres.url` properties with different values in the pom?

- `vertx-reactive:postgresql:///hibernate_orm_test`
- `vertx-reactive:postgresql://:5431/hibernate_orm_test`

Comment (gsmet) [reply extensions/reactive-pg-client/deployment/pom.xml:20]: The first one is when Docker is not used: we use the default port of the global instance.

The second one is for Docker as we put it on another port.

**Comment by vsevel** [comment]
perfect
I suppose I need to wait for you to merge into master,and I will rebase from there?

**Comment by gsmet** [comment]
@vsevel merged! if you have any issues with adding your tests to the deployment modules, please ping me, I'll help!

**Comment by vsevel** [comment]
I will give it a try tonight. thanks

**Pull request merged**